package.json

```diff
@@ -408,7 +414,7 @@
     "build-watch:cljs": "yarn && shadow-cljs watch app",
     "build-watch:js": "yarn && NODE_OPTIONS=--max-old-space-size=8192 rspack serve --watch",
     "build:cljs": "yarn && yarn build-pure:cljs",
-    "build:js": "yarn && rspack build",
+    "build:js": "concurrently -n 'install,build' -c 'blue,green' 'yarn install' 'rspack build'",
```

Comment:
Why?
I think this was sequential on purpose in the past because you cannot build if no libraries are installed.